website/project/model.py

```diff
@@ -4074,7 +4073,7 @@ def _on_complete(self, user):
         parent_registration.registered_from.add_log(
             action=NodeLog.EMBARGO_APPROVED,
             params={
-                'node': parent_registration._id,
+                'node': parent_registration.registered_from_id,
```

Comment:
same as above, needs to keep the registered node in the log params.

e.g. `'registration_node': self._id,`